packages/api/src/external/carequality/document/process-outbound-document-query-resps.ts

```diff
@@ -37,18 +36,21 @@ export async function processOutboundDocumentQueryResps({
   cxId: string;
   outboundDocumentQueryResps: OutboundDocumentQueryResp[];
 }): Promise<void> {
-  if (!iheGateway) return;
-  const { log } = out(`CQ query docs - requestId ${requestId}, M patient ${patientId}`);
+  const { log } = out(`CQ DR - requestId ${requestId}, M patient ${patientId}`);
 
-  const { organization } = await getPatientWithDependencies({ id: patientId, cxId });
+  const interrupt = buildInterrupt({ requestId, patientId, cxId, log });
+  if (!iheGateway) return interrupt(`IHE GW not available`);
+  if (!lambdaName) return interrupt(`IHE DR lambda not available`);
+  if (!(await isCQDirectEnabledForCx(cxId))) return interrupt(`CQ disabled for cx ${cxId}`);
```

Comment:
If the IHE GW is not available or CQ not enabled for this cx at this point (already did DQ and is triggering DR), fail, warn, log, scream 😱